src/services/request-presentation-service.ts

```diff
@@ -25,6 +25,18 @@ export class RequestPresentationService implements IRequestPresentationService {
     switch (request.status) {
       case RequestStatus.COMPLETED: {
         const anchor = await this.anchorRepository.findByRequest(request)
+        // TODO: This is a workaround, fix in CDB-2192
+        const anchorCommit = {
+          cid: anchor ? anchor.cid : request.cid,
+          content: {
```

Comment:
wow didn't even realize we returned the anchor commit content.  We can probably investigate removing this `content` field from the responses entirely (later)